File: python/tabla de la verdad.py
def tabla_de_verdad():
    print("A\tB\tA AND B\tA OR B\tNOT A\tNOT B")
    print("-" * 40)

    # Iterar sobre todas las combinaciones de A y B
    for A in [True, False]:
        for B in [True, False]:
            and_result = A and B
            or_result = A or B
            not_A = not A
            not_B = not B
            
            # Imprimir los resultados
            print(f"{A}\t{B}\t{and_result}\t{or_result}\t{not_A}\t{not_B}")

# Llamar a la función
tabla_de_verdad()

File: javaScript/tabla de  la verdad.js
function tablaDeVerdad() {
    console.log("A\tB\tA AND B\tA OR B\tNOT A\tNOT B");
    console.log("-".repeat(40));

    // Iterar sobre todas las combinaciones de A y B
    for (let A of [true, false]) {
        for (let B of [true, false]) {
            const andResult = A && B;
            const orResult = A || B;
            const notA = !A;
            const notB = !B;

            // Imprimir los resultados
            console.log(`${A}\t${B}\t${andResult}\t${orResult}\t${notA}\t${notB}`);
        }
    }
}